views/managing-officer-corporate.html

```diff
@@ -0,0 +1,303 @@
+        fieldset: {
+          legend: {
+            text: "Is their correspondence address the same as the principal or registered office address?",
+            isPageHeading: false,
+            classes: "govuk-fieldset__legend--m"
+          }
+        },
+        items: [
+          {
+            value: "yes",
+            text: "Yes"
+          },
+          {
+            value: "no",
+            text: "No",
+            conditional: {
+              html: serviceAddressHtml
+            }
+          }
+        ]
+      }) }}
+
+      {% endcall %}
+
+      {{ govukInput({
+        label: {
+          text: "Where is the corporate managing officer registered?",
+          classes: "govuk-label--m",
+          isPageHeading: false
+        },
+        id: "corporateManagingOfficerWhereRegistered",
+        name: "corporateManagingOfficerWhereRegistered"
+      }) }}
+
+      {{ govukInput({
+        label: {
+          text: "What is its legal form?",
+          classes: "govuk-label--m",
+          isPageHeading: false
+        },
+        hint: {
+          text: "For example, limited company or partnership."
+        },
+        id: "corporateManagingOfficerLegalForm",
+        name: "corporateManagingOfficerLegalForm"
+      }) }}
+
+      {{ govukInput({
+        label: {
+          text: "What is its governing law?",
+          classes: "govuk-label--m",
+          isPageHeading: true
+        },
+        hint: {
+          text: "This is the law that the entity operates under."
+        },
+        id: "corporateManagingOfficerLegalAuthority",
+        name: "corporateManagingOfficerLegalAuthority"
+      }) }}
+
+
+      {{ govukInput({
+        label: {
+          text: "What is its registration number?",
+          classes: "govuk-label--m",
+          isPageHeading: false
+        },
+        classes: "govuk-!-width-one-half",
+        id: "corporateManagingOfficeRegistrationNumber",
+        name: "corporateManagingOfficeRegistrationNumber"
+      }) }}
+
+      {{ govukDateInput({
+        id: "startDate",
+        namePrefix: "startDate",
+        fieldset: {
+          legend: {
+            text: "When did it become a managing officer for the overseas entity?",
+            isPageHeading: false,
+            classes: "govuk-fieldset__legend--m"
+          }
+        },
+        hint: {
+          text: "For example, 27 3 2007"
+        }
+      }) }}
+
+      <h2 class="govuk-heading-m">What information is shown on the online register</h2>
+      <p>All information provided on this screen about the corporate managing owner will be available on the public register.</p>
+
+      {{ govukButton({
+        text: "Add another corporate managing officer",
+        href: "",
+        classes: "govuk-button--secondary govuk-!-margin-top-6"
+      }) }}
+
+      {{ govukButton({
+        text: "I do not need to add any more corporate managing officers",
+        href: "/register/check-answers"
```

Comment:
button needs a submit attribute - see https://github.com/companieshouse/overseas-entities-web/blob/main/views/beneficial-owner-gov.html#L311

Reply:
Wont the other button need an id as well so the automated tests know the difference between them?

Reply:
yes, although i didn't add one on my template (for the first button) not sure what the the id would be as submit would be used for the 'submit' page (second button)

Reply:
i think Sam is right, we should add an id attribute for the first button, I'll do the same on my page. I'm going to add an id of 'add' on mine, if you do the same here then they'll be consistent.